collectors/health_test.go

```diff
@@ -31,23 +31,6 @@ func TestClusterHealthCollector(t *testing.T) {
 	}{
 		{
 			input: `
-{
```

Comment:
Why did we need to take these out?

Reply:
These test cases failed becase the logic for setting the gauges based on `pgmap` would set all the pg state related gauges to 0, failing the tests. If `pgmap` is always present, then we're good. If there's an edge case where only summary is present, then I need to adjust this approach.

Reply:
To be clear, I've tested this out and it seems to work fine. Just wanted to make sure that the exporter wasn't setting those gauges redundantly for a reason.

Reply:
Gotcha. Sounds good to me!